client/http_sender_test.go

```diff
@@ -63,7 +63,7 @@ func startTestHTTPServer(handler http.Handler) (*httptest.Server, string) {
 // TestHTTPSenderSend verifies sending posts.
 func TestHTTPSenderSend(t *testing.T) {
 	server, addr := startTestHTTPServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.Method != "POST" {
+		if r.Method != util.PostMethod {
```

Comment:
I'd be more excited about this type of change if it was actually fixing a bug. As it is, this is less readable than simply typing out `"POST"`.